diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9efd451fb..05066be82 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -643,26 +643,6 @@ target_optional_compile_definitions(torrent-rasterbar PUBLIC FEATURE NAME mutabl
 target_optional_compile_definitions(torrent-rasterbar PUBLIC FEATURE NAME streaming DEFAULT ON
 	DESCRIPTION "Enables support for piece deadline" DISABLED TORRENT_DISABLE_STREAMING)

-find_public_dependency(Iconv)
-if(MSVC)
-	set(iconv_package_type OPTIONAL)
-else()
-	set(iconv_package_type RECOMMENDED)
-endif()
-
-set_package_properties(Iconv
-	PROPERTIES
-		URL "https://www.gnu.org/software/libiconv/"
-		DESCRIPTION "GNU encoding conversion library"
-		TYPE ${iconv_package_type}
-		PURPOSE "Convert strings between various encodings"
-)
-
-if(Iconv_FOUND)
-	target_compile_definitions(torrent-rasterbar PUBLIC TORRENT_USE_ICONV)
-	target_link_libraries(torrent-rasterbar PRIVATE Iconv::Iconv)
-endif()
-
 find_public_dependency(OpenSSL)
 set_package_properties(OpenSSL
 	PROPERTIES
